package.json

```diff
@@ -6,7 +6,8 @@
   "scripts": {
     "test": "node_modules/.bin/standard --verbose | snazzy",
     "build": "rimraf dist/ && babel ./**/*.js --out-dir dist/ --ignore ./node_modules,./.babelrc,./package.json,./npm-debug.log --copy-files",
-    "clean:data": "./mvnw spring-boot:run -DprojectCleanUp=true -DprojectSetUp=false -DchannelsImport=false -DcategoryImport=false -DcatalogImport=false -DreserveInStore=false -DordersImport=false -Dctp.projectKey=$npm_package_config_projectKey -Dctp.clientId=$npm_package_config_clientId -Dctp.clientSecret=$npm_package_config_clientSecret -Dctp.authUrl=$npm_package_config_authUrl -Dctp.apiUrl=$npm_package_config_apiUrl -Dctp.protocol=https",
+    "preclean:data": "npm run build",
+    "clean:data": "node dist/lib/clear.js --projectKey $npm_package_config_projectKey --clientId $npm_package_config_clientId --clientSecret $npm_package_config_clientSecret --apiUrl $npm_package_config_apiUrl --authUrl $npm_package_config_authUrl",
```

Comment:
I think it's unnecessary user confusion to call the script "clean" while calling the implementation "clear".  Does that have a reason?

Reply:
It has no reason, it's just some stupid mistake :D 
I will rename it all just to 'clean'